internals/generators/route/index.js

```diff
@@ -39,6 +50,11 @@ module.exports = {
 
       return 'path is required';
     },
+  }, {
+    type: 'confirm',
+    name: 'useSagas',
+    default: sagasExists(componentName) ? true : false,
+    message: 'Do you want to use sagas?',
```

Comment:
How about changing message to:
> Do you want sagas for asynchronous flows? (e.g. fetching data)

This is the sagas prompt from the container generator.